src/components/Search/SearchInput.scss

```
@@ -1,7 +1,6 @@
-@use "~@redhat-cloud-services/frontend-components-utilities/styles/_all";
+@use '~@redhat-cloud-services/frontend-components-utilities/styles/_all';
```

Stylelint (17.14.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)